README.md

```diff
@@ -1,70 +1,72 @@
+# Happy Hacking!!
+Example Final Product:
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+[https://s3.us-east-2.amazonaws.com/files.kenzie.academy/frontend-q2/github-card.mp4](https://s3.us-east-2.amazonaws.com/files.kenzie.academy/frontend-q2/github-card.mp4)
 
-## Learn More
+**Stretch Goals**
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+* Add a new section to the page that contains a form with an input box. The input box should ask for a Github username. After the user types in a Github username and submits the form, then it should display a card (should look the same as the one created in Part 3) with that users information. The form can be resubmitted anytime with a new username and it should continue to display the card but with the new users information. Note: for this bonus feature, you should try to re-use the jsx you already wrote for the card from Part 3. You can easily re-use this jsx by creating a named component containing that jsx. A possible name could be "GithubCard".
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+===
 
-### Code Splitting
+Author: Gavin Ferise
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+References: 
+* https://upmostly.com/tutorials/how-to-use-the-usecontext-hook-in-react
+* https://react-bootstrap.github.io/components/forms/#forms-validation
+* Regex in NameForm.js based on https://github.com/shinnn/github-username-regex
```

Comment:
This is really really really cool :D :D :D : D Super proud of you for creating your own readme! I know it might be silly, but you'd be surprised how important it is and how few people do it ;D